test/e2e/framework/pod/wait.go

```diff
@@ -238,7 +238,7 @@ func WaitForPodsRunningReady(c clientset.Interface, ns string, minPods, allowedN
 		if !ignoreNotReady {
 			return errorBadPodsStates(badPods, desiredPods, ns, "RUNNING and READY", timeout, lastAPIError)
 		}
-		e2elog.Logf("Number of not-ready pods (%d) is below the allowed threshold (%d).", notReady, allowedNotReadyPods)
+		e2elog.Logf("Number of not-ready pods (%d) is above the allowed threshold (%d).", notReady, allowedNotReadyPods)
```

Comment:
at this line ignoreNotReady is true which means notReady <= allowedNotReadyPods or its below?
am i getting confused?

Reply:
https://github.com/debdeepsanyal/kubernetes/blob/725aa23bb1a53a6a1b6c36f03bec35fa58295c87/test/e2e/framework/pod/wait.go#L238-L241